### PR TITLE
Add CASE

### PR DESCRIPTION
**Pull request by SebConejo**
Add CASE, an open-source and frameworkless tool allowing to build CRUD web applications in minutes.

### CODE REVIEW

**Review by yenaingtun-dev — APPROVED** [review]

**Comment by SebConejo** [comment]
Hi @yenaingtun-dev , I saw you approved this pr 4 weeks ago. We updated the project name. Once you approve it, do you know when the PR will be merged ? 

Thanks you,

File: data.json
Comment (sammyhori) [inline data.json:2102]:
```suggestion
          "label": "good first issue",
```

**Comment by sammyhori** [comment]
Looks good otherwise :+1:

**Review by sammyhori — APPROVED** [review]

**Pull request merged**
